sql/src/main/java/io/crate/planner/node/management/ExplainPlan.java

```diff
@@ -43,10 +47,16 @@ public ExplainPlan(Plan subPlan) {
 
     @Override
     public UUID jobId() {
-        return subPlan.jobId();
+        return subExecutionPlan.jobId();
     }
 
-    public Plan subPlan() {
-        return subPlan;
+    public ExecutionPlan subPlan() {
+        return subExecutionPlan;
```

Comment:
This seems a bit trappy, since this is initialized lazy.

Couldn't we change the `ExplainTask` to take an ExecutionPlan as argument, and then have the `TaskCollectingVisitor.visitExplainPlan` call build on the Plan?